cleanlab/datalab/internal/issue_manager/duplicate.py

```diff
@@ -98,13 +98,19 @@ def find_issues(
             knn_graph = knn.kneighbors_graph(mode="distance")
         N = knn_graph.shape[0]
         nn_distances = knn_graph.data.reshape(N, -1)[:, 0]
-        scores = np.tanh(nn_distances)
-        self.near_duplicate_sets = self._neighbors_within_radius(knn_graph, self.threshold)
+        median_nn_distance = max(
+            np.median(nn_distances), np.finfo(np.float_).eps
+        )  # avoid threshold = 0
```

Comment:
```suggestion
        )  # avoid zero median in denominator
```

no discussion of threshold here yet

Reply:
That's from the previous PR. 

It's unrelated to the temperature being set.

Reply:
It's actually still relevant to the `self.threshold`. Which is already set at this point in the method.

Reply:
I'd keep it as is.